.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
Hmm, seems like this should be removed from the PR. If you have VSCode settings you want locally, you can setup a local `gitignore`.